ush/ioda/bufr2ioda/run_bufr2ioda.sh

```diff
@@ -51,7 +51,8 @@ export PYTHONPATH=${PYIODALIB}:${PYTHONPATH}
 #---------------------------
 #----- python and json -----
 # first specify what observation types will be processed by a script
-BUFR_py="satwind_amv_goes"
+#BUFR_py="satwind_amv_goes"
+BUFR_py="adpupa_prepbufr"
```

Comment:
Can BUFR_py only take one data type?

Reply:
`BUFR_py` can be a list.  `run_bufr2ioda.sh` uses `BUFR_py` in a shell loop.  `run_bufr2ioda.py` also contains `BUFR_py`.  I successfully executed `run_bufr2ioda.py` from `prepatmiodaobs` with `BUFR_py=["satwind_amv_goes", "adpupa_prepbufr"]`.

Reply:
Yes this list should be inclusive in the develop branch